README.md

```diff
@@ -0,0 +1,425 @@
+Running Tests as VM Jobs
+========================
+
+This repository drives the OSG Software nightly tests. Whenever updating [osg-run-tests](osg-run-tests), make sure to update the copy in `/usr/bin/` on `osghost`.
+
+Setting up credentials
+----------------------
+
+**Note:** This one time setup must be performed before submitting any test jobs.
+
+1.  Contact Brian L. or Tim C. to have an account created on `osghost.chtc.wisc.edu`
+2.  SSH to `osghost` (you will need to do this from the UW-Madison network) and create a password-less SSH key:
+
+        [user@client ~]$ ssh-keygen -C 'VMU test result upload key for <USER>@osghost' -f ~/.ssh/test_result_upload.key
+
+3.  Contact BrianL or MatS with your public key. They will add your public key to the `authorized_keys` file of the `cndrutil` user on `ingwe`.
```

Comment:
"MatS" looks too much like "Mats". Just remove the "S" - plain "Mat" is what's used elsewhere.